client/server/middleware/unsupported-browser.js

```diff
@@ -18,13 +18,32 @@ function isSupportedBrowser( req ) {
 	} );
 }
 
+// We don't want to redirect some of our public landing pages, so we include them
+// here.
+function allowPath( path ) {
+	// Strip leading '/'.
+	let parsedPath = path.replace( /^\//, '' );
+	const possiblePathLocales = [ 'en', ...config( 'magnificent_non_en_locales' ) ];
+	for ( const locale of possiblePathLocales ) {
+		// Strip leading locale (e.g. 'es/')
+		if ( parsedPath.startsWith( locale ) ) {
```

Comment:
This matching is implemented a bit carelessly I would say. It will match also `/design` and `/devdocs` and the `parsedPath` becomes `sign` or `vdocs`, that is further matched against `start` or `new`...

The `allowedPaths` will also accept `/startups` or `/newport` although it shouldn't... Although there probably isn't any Calypso URL that would get mismatched, we can do better, by also checking the `/` characters at expected places.

Reply:
Thanks! Fixed: #56737